src/bundle/Resources/public/js/scripts/admin.location.bookmark.js

```diff
@@ -15,14 +16,31 @@
     const isCurrentLocation = (locationId) => {
         return parseInt(locationId, 10) === currentLocationId;
     };
+    const setBookmarkWrapperCheckedClass = (bookmarked) => {
+        const wrapperClassList = document.querySelector(SELECTOR_BOOKMARK_WRAPPER).classList;
+        const bookmarkCheckedClass = 'ez-add-to-bookmarks--checked';
+
+        if (bookmarked) {
+            wrapperClassList.add(bookmarkCheckedClass);
+        } else {
+            wrapperClassList.remove(bookmarkCheckedClass);
+        }
+    };
     const updateBookmarkForm = (event) => {
         const { bookmarked, locationId } = event.detail;
 
         if (isCurrentLocation(locationId)) {
             updateBookmarkCheckbox(bookmarked);
+            setBookmarkWrapperCheckedClass(bookmarked);
         }
     };
+    const updateBookmarkWrapperClass = (event) => {
```

Comment:
`updateBookmarkCheckedState`

Reply:
I think that `updateBookmarkCheckedState ` suggests that the input state is being updated, when it is not.

Reply:
Maybe, `updateBookmarkIconState`?